cypress-tests/cypress/e2e/PaymentUtils/Paybox.js

```diff
@@ -0,0 +1,142 @@
+const successfulNo3DSCardDetails = {
+    card_number: "1111222233334444",
+    card_exp_month: "05",
+    card_exp_year: "27",
+    card_holder_name: "joseph Doe",
+    card_cvc: "222",
+};
+
+export const connectorDetails = {
+    card_pm: {
+        PaymentIntent: {
+            Request: {
+                payment_method: "card",
+                payment_method_data: {
+                    card: successfulNo3DSCardDetails,
+                },
+                currency: "EUR",
+                customer_acceptance: null,
+                setup_future_usage: "on_session",
+            },
+            Response: {
+                status: 200,
+                body: {
+                    status: "requires_payment_method",
+                },
+            },
+        },
+        No3DSManualCapture: {
+            Request: {
+                payment_method: "card",
+                payment_method_data: {
+                    card: successfulNo3DSCardDetails,
+                },
+                currency: "EUR",
```

Comment:
please remove `currency` field from confirm call. here and everywhere except `PaymentIntent.